tcms/testcases/tests/test_views.py

```diff
@@ -179,6 +179,32 @@ def setUpTestData(cls):
         user_should_have_perm(cls.tester, 'testcases.add_testcase')
         user_should_have_perm(cls.tester, 'testcases.view_testcase')
 
+    def test_notify_formset_invalid(self):
+
+        data = self.data.copy()
```

Comment:
The place where you have copied this from contains an important comment as to why we are doing it this way. Copy the comment here as well b/c it is not immediately obvious why this test works like that.

Reply:
Done!